src/components/Login/Login.js

```diff
@@ -159,7 +159,7 @@ const Login = ({ intl, location, dependencies: { dopplerLegacyClient, sessionMan
         <p className="content-subtitle">{_('login.enter_doppler_sub')}</p>
         <p className="content-subtitle">
           {_('login.you_want_create_account')}{' '}
-          <Link to="/signup" className="link--title">
+          <Link to="/signup">
```

Comment:
Me parece que esto no es un breaking change.

Reply:
Menos para la webapp, donde el cliente es un usuario humano (en general más tolerante a los cambios que una máquina)

Reply:
Que le pondrías vos? Que es un refactor?

Reply:
If we are eliminanting redundant classes we are not changing nothing substantial, I woudl say this is a chore.